tests/two_d_primitive_tests.cpp

```diff
@@ -109,23 +114,23 @@ void TwoDPrimitiveTests::Initialize() {
 void TwoDPrimitiveTests::Test(const BlitTest& test) {
   host_.PrepareDraw(0xFF440011);
 
+  auto p = pb_begin();
 
-//  uint32_t clip_w = host_.GetFramebufferWidth();
-//  uint32_t clip_h = host_.GetFramebufferHeight();
-//
-//  ImageBlit(test.blit_operation, test.beta, image_src_dma_ctx_.ChannelID,
-//            11,  // DMA channel 11 - 0x1117
-//            test.buffer_color_format, image_pitch_, 4 * host_.GetFramebufferWidth(), 0, START_X, START_Y, 0,
-//            DESTINATION_X, DESTINATION_Y, SOURCE_WIDTH, SOURCE_HEIGHT, clip_x, clip_y, clip_w, clip_h);
+  p = pb_push1_to(SUBCH_CLASS_42, p, NV04_CONTEXT_SURFACES_2D_SET_DMA_IMAGE_DST, /*dst_dma_ctx_.ChannelID*/ 9);
```

Comment:
@abaire what does 9 represent here?

Reply:
DMA channel 9 (see https://github.com/XboxDev/nxdk/blob/master/lib/pbkit/pbkit.c#L3102 and https://github.com/XboxDev/nxdk/blob/master/lib/pbkit/pbkit.c#L1859)
I'd strongly suggest putting in a symbolic name instead of the quick hack `9` literal I used since the reader would have to be quite familiar with pbkit to know this)